src/components/label/label.scss

```diff
@@ -65,7 +65,3 @@ ion-label[floating] {
 
   max-width: 100%;
 }
-
```

Comment:
This CSS rule was forcing the floating label to always "float". It was needed because floating labels only float when the ion-item has `input-has-value` class. Now that it's being set by the ion-select, there's no need to force it.